File: sdks/insuretech-go-sdk/pkg/models/login_response.go
package models


// LoginResponse represents a login_response
type LoginResponse struct {
	AccessToken string `json:"access_token,omitempty"`
	AccessTokenExpiresIn int `json:"access_token_expires_in,omitempty"`
	CsrfToken string `json:"csrf_token,omitempty"`
	MfaMethod string `json:"mfa_method,omitempty"`
	MfaRequired bool `json:"mfa_required,omitempty"`
	MfaSessionToken string `json:"mfa_session_token,omitempty"`
	PasswordChangeRequired bool `json:"password_change_required,omitempty"`
	RefreshToken string `json:"refresh_token,omitempty"`
	RefreshTokenExpiresIn int `json:"refresh_token_expires_in,omitempty"`
	SessionId string `json:"session_id,omitempty"`
	SessionToken string `json:"session_token,omitempty"`
	SessionType string `json:"session_type,omitempty"`
	User *User `json:"user,omitempty"`
	UserId string `json:"user_id,omitempty"`
}

File: Android/LabAidInsureTech/app/src/main/java/com/labaid/insuretech/data/model/login/LoginResponse.kt
package com.labaid.insuretech.data.model.login


import com.google.gson.annotations.SerializedName

data class LoginResponse(
    @SerializedName("user_id")
    val userId: String?,
    @SerializedName("access_token")
    val accessToken: String?,
    @SerializedName("refresh_token")
    val refreshToken: String?,
    @SerializedName("access_token_expires_in")
    val accessTokenExpiresIn: Int?,
    @SerializedName("refresh_token_expires_in")
    val refreshTokenExpiresIn: Int?,
    @SerializedName("user")
    val user: User?,
    @SerializedName("error")
    val error: Error?
)